packages/react/src/themes/base/components/Loader/loaderStyles.ts

```diff
@@ -52,8 +52,6 @@ export default {
       borderStyle: 'solid',
       borderWidth: v.borderSizes[p.size],
 
-      boxSizing: 'border-box',
-
```

Comment:
Not sure about this, it's defined in Base theme

Reply:
My mistake, thought it was in team's theme. Will revert it.